deepmd/pt/model/descriptor/se_t_tebd.py

```
@@ -441,12 +441,14 @@ def forward(
             The smooth switch function. shape: nf x nloc x nnei
 
         """
+        # cast the input to internal precsion
+        extended_coord = extended_coord.to(dtype=self.prec)
         del mapping
         nframes, nloc, nnei = nlist.shape
         nall = extended_coord.view(nframes, -1).shape[1] // 3
```

Ruff

Local variable `nall` is assigned to but never used

Remove assignment to unused variable `nall`

(F841)